### PR TITLE
Added Crystal17 to the Intel stack

### PR DESCRIPTION
**Pull request by batatasfritas**

### CODE REVIEW

File: configuration/modules.yaml [hunk 1]
```diff
@@ -70,6 +70,7 @@ modules:
       'cfdplusplus',
       'cmake%gcc@4.8.5',
       'comsol',
+      'crystal',
```
Comment (rmsds) [inline configuration/modules.yaml]: The package name is `crystal17` ;-)

File: configuration/modules.yaml [hunk 2]
```diff
@@ -310,6 +311,9 @@ modules:
         prepend_path:
           MATLABPATH: '${PREFIX}/mli'
 
+    crystal:
```
Comment (rmsds) [inline configuration/modules.yaml]: Please fix the package name.

File: configuration/packages.yaml [hunk 3]
```diff
@@ -136,6 +136,11 @@ packages:
   # Concretization preferences
   ####
 
+  crystal:
```
Comment (rmsds) [inline configuration/packages.yaml]: Please fix the package name.

**Review by rmsds — CHANGES_REQUESTED** [review]
The package name is wrong in a few places.

File: configuration/modules.yaml (hunk 1 above)
Comment (batatasfritas) [reply configuration/modules.yaml]: Yeah... initially I had named the package crystal and then changed to crystal17 and I think all things got mixed up in my head. Sorry about that and thanks for noticing it.

**Comment by batatasfritas** [comment]
OK, there was a minor problem (scitasbuild was not part of the crystal-soft group because of the cache) but that's not automagically fixed.

**Review by rmsds — APPROVED** [review]

**Pull request merged**